edge/pkg/metamanager/process.go

```diff
@@ -549,12 +545,13 @@ func (m *metaManager) syncPodStatus() {
 			klog.Errorf("unmarshal podstatus[%s] failed, content[%s]: %v", v.Key, v.Value, err)
 			continue
 		}
-		content = append(content, podStatus)
+		contents[namespaceParsed] = append(contents[namespaceParsed], podStatus)
+	}
+	for namespace, content := range contents {
+		msg := model.NewMessage("").BuildRouter(MetaManagerModuleName, GroupResource, namespace+constants.ResourceSep+model.ResourceTypePodStatus, model.UpdateOperation).FillBody(content)
+		sendToCloud(msg)
+		klog.Infof("sync pod status successful for namespaces %s, %s", namespace, msgDebugInfo(msg))
```

Comment:
We can set the log level here to avoid to many logs, `klog.V(3).Infof`.
Ref: https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/status/status_manager.go#L578